src/Domain/Donation.php

```diff
@@ -24,6 +24,17 @@
  */
 class Donation extends SalesforceWriteProxy
 {
+    public const STATUSES = [
+        'Pending',
+        'Collected',
+        'Paid',
+        'Cancelled',
+        'Refunded',
+        'Failed',
+        'Chargedback',
+        'RefundingPending',
+        'PendingCancellation',
+    ];
```

Comment:
I've checked the live database to make sure this isn't an incomplete list:

```SELECT DISTINCT donationStatus from Donation; ```

returns

```
Collected	
Paid	
Cancelled	
Pending	
Refunded	
```